application/web/web.js

```diff
@@ -1,3 +1,22 @@
 window.onload = () => {
-    
+    createSidebar();
+};
+
+function createSidebar() {
+
+     const journalEntries = document.querySelectorAll('.journal-entry');
+
+     journalEntries.forEach(entry => {
+
+        const faIcon = entry.querySelector('.fa.fa-ellipsis-v');
+
+        entry.addEventListener('mouseover', () => {
+            faIcon.style.display = 'block';
+        });
+
+        entry.addEventListener('mouseout', () => {
+            faIcon.style.display = 'none';
+        });
```

Comment:
Just a suggestion :)

Might be worth looking up 'event delegation'

Here is a code snippet: 

`document.querySelector('#parent-element').addEventListener('click', function(event) {
  const targetElement = event.target.closest('.child-element');
  if (targetElement) {
    // Handle the event for the child element
    console.log('Child element clicked:', targetElement);
  }
});`